msteams-platform/webhooks-and-connectors/how-to/connectors-using.md

```diff
@@ -160,82 +160,43 @@ The following steps use [cURL](https://curl.haxx.se/). We assume that you have t
    curl.exe -H "Content-Type:application/json" -d "{'text':'Hello World'}" <YOUR WEBHOOK URL>
    ```
 
-1. If the POST succeeds, you should see a simple **1** output by `curl`.
-1. Check the Microsoft Team client. You should see the new card posted to the team.
+    > [!NOTE]
+    > If the POST succeeds, you must see a simple **1** output by `curl`.
 
-### Post a message to the webhook using PowerShell
+1. Check the Microsoft Teams client for the new card posted.
 
-The following steps use PowerShell. We assume that you have this installed and are familiar with its basic usage.
+# [PowerShell](#tab/PowerShell)
+
+ Prerequisite: Installation of PowerShell and familiarization with its basic usage.
+
+**To post a message to the webhook with PowerShell**
 
 1. From the PowerShell prompt, enter the following command:
 
    ```powershell
    Invoke-RestMethod -Method post -ContentType 'Application/Json' -Body '{"text":"Hello World!"}' -Uri <YOUR WEBHOOK URL>
    ```
 
-1. If the POST succeeds, you should see a simple **1** output by `Invoke-RestMethod`.
-1. Check the Microsoft Teams channel associated with the webhook URL. You should see the new card posted to the channel.
+    > [!NOTE]
+    > If the POST succeeds, you must see a simple **1** output by `Invoke-RestMethod`.
 
-- [Include two icons](../../concepts/build-and-test/apps-package.md#app-icons).
-- Modify the `icons` portion of the manifest to refer to the file names of the icons instead of URLs.
+1. Check the Microsoft Teams channel associated with the webhook URL. You can see the new card posted to the channel. Before you use the connector to test or publish your app, you must do the following:
 
-The following manifest.json file contains the basic elements needed to test and submit your app:
-
-> [!NOTE]
```

Comment:
I think this entire section is misplace. Could you please check and remove this? This is already present here: https://review.docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-creating?branch=pr-en-us-3657#including-the-connector-in-your-manifest